### PR TITLE
passthrough_hp: Fix clone-fd option

### PR DESCRIPTION
**Pull request by SteveY4ng**
The clone-fd option is set valued but not used in the context. Use it in the code.

### CODE REVIEW

**Review by bsbernd — APPROVED** [review]
Thanks, yeah I had forgotten it that. Actually I think have a patch like this in one of the branches - looks like I forgot to submit it. Thanks for fixing it!

**Comment by bsbernd** [comment]
Hmm, I wonder why it is stuck at "Waiting for status to be reported ". @Nikratio any idea?

**Comment by SteveY4ng** [comment]
> Hmm, I wonder why it is stuck at "Waiting for status to be reported ". @Nikratio any idea?

Hello! I think the reason is that  the maintainers need to run some workflows on pull requests from contributors that require approval. I don't have the write access to this repository. And the workflows need to be started by the maintainers manually. 
You can see this [Link](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks) for more information.

**Pull request merged**
